tests/api.test.js

```diff
@@ -122,10 +273,11 @@ describe('Sample Test', () => {
     expect(res.body.err).toBe("Invalid ID");
     done();
   });
-  it('Close demand', async (done) => {
+
+  // Need to close demand before trying to get closed demands, same goes for getting a opened demands
```

Comment:
Fix the typo "a" in "same goes for getting a opened demands".

Reply:
trocado para "an"

Reply:
No caso a mudança seria para "same goes for getting opened  demands"

Reply:
ok